Source/WebCore/Modules/indexeddb/IDBOpenDBRequest.cpp

```diff
@@ -196,7 +196,7 @@ void IDBOpenDBRequest::requestCompleted(const IDBResultData& data)
     if (isContextStopped()) {
         switch (data.type()) {
         case IDBResultType::OpenDatabaseSuccess:
-            connectionProxy().abortOpenAndUpgradeNeeded(data.databaseConnectionIdentifier(), IDBResourceIdentifier::emptyValue());
+            connectionProxy().abortOpenAndUpgradeNeeded(data.databaseConnectionIdentifier(), IDBResourceIdentifier(connectionProxy(), *this));
```

Comment:
This seems not quite right since the function `IDBConnectionProxy::abortOpenAndUpgradeNeeded` asks for a transaction identifier, and we pass a request identifier. Though the result might be expected, as `IDBStorageRegistry` won't find a transaction with a request identifier, it can be hard to reason about the code later (that a request identifier is used as place holder for a finished transaction).

We should probably change the parameter to `std::optional<IDBDatabaseIdentifier>&& databaseIdentifier` and use std::nullopt here